frameworks/g-cloud-6/questions/services/lotName.yml

```diff
@@ -0,0 +1,2 @@
+question: 'Service type'
```

Comment:
This is very similar to the `serviceTypes` question - why not something like `Lot name` for these dummy questions?